prow/plugins/approve/approve.go

```diff
@@ -143,17 +142,18 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 }
 
 func handleGenericCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
+	opts := pc.PluginConfig.Approve.RepoOptions(ce.Repo.Owner.Login, ce.Repo.Name)
```

Comment:
The `approve` plugin should use `BranchOptions` not repo options since it operates on PRs which are always associated with a base branch.